edgePy/util.py

```diff
@@ -29,8 +29,8 @@ def getLogger(
     Examples:
         >>> from edgePy.util import getLogger
         >>> log = getLogger(name="script")
-        >>> log.info('This is the your DGElist.')
-        [INFO | [180904 14:07:30 | script | DGEList | line 1]: This is the your DGElist.
+        >>> log.info('This is your DGElist.')
+        ...
```

Comment:
Try this:

```python
    Examples:
        >>> from edgePy.util import getLogger
        >>> log = getLogger(name="script")
        >>> log.info('This is your DGElist.')
        [INFO | [... | script | DGEList | line 1]: This is your DGElist.
```

Reply:
@clintval that keeps failing for me locally when I try to run with the doctest module using t`he `doctest.ELLIPSIS` option flag. Perhaps that's by design?